lib/hexo/load_config.js

```diff
@@ -30,10 +30,27 @@ module.exports = async ctx => {
   validateConfig(ctx);
 
   ctx.config_path = configPath;
-
+  // Trim multiple trailing '/'
   config.root = config.root.replace(/\/*$/, '/');
+  // Remove any trailing '/'
   config.url = config.url.replace(/\/+$/, '');
 
+  // Deprecated: config.external_link boolean option will be removed in future
+  if (config.external_link === true) {
```

Comment:
suggest to handle `typeof config.external_link === 'undefined'` too.

Reply:
@curbengh Test case for `undefined` is added in https://github.com/hexojs/hexo/pull/4414/commits/e22a3271721298f493a83be763a8d109508409a2.